_posts/2020-03-15-Sharezone-In-Review-March-Part-1.md

```diff
@@ -0,0 +1,88 @@
+Es bleiben schnell Aufgaben liegen, die zwar wichtig, aber nicht direkt dringend sind - Beispielsweise eine automatisierte Erstellung von Demo-Accounts, die uns viel Zeit und Arbeit ersparen und Schulen besser addressieren würde (falls diese direkt selbst Demo-Accounts erstellen könnten). 
+
+Das ganze Thema "Sales/Marketing" ist etwas, wo wir noch viel mehr tun sollten, aber es bisher noch nicht richtig priorisiert haben. Alleine unsere Website ist ja kaum mehr als eine Seite an Inhalt 😁   
+Gleichzeitig ist das aber auch einer der Gründe, warum wir Mitarbeiter suchen: Mit mehr Leuten, die uns dringende Aufgaben abnehemen, können wir die wirklich wichtigen Aufgaben erledigen und außerdem so auch mehr **am** Unternehmen und weniger **im** Unternehmen arbeiten.
+
+Um den ersten Schritt in die Richtung von Automatisierung zu gehen, haben wir jetzt eine interne Anleitung erstellt. Diese beschreibt genaustens was für Inhalte erstellt werden müssen und liefert entsprechendene Vorlagen. 
+Der zweiten Schritt in Zukunft wäre dann genau diese Anleitung zu automatisieren.    
+Manuell machen, dokumentieren, automatisieren.    
+
+## Ideen für die Zukunft
+Mit weiterer Entwicklung des Pitch-Decks haben wir uns natürlich auch nochmal Gedanken über die Vision von Sharezone in der Zukunft gemacht.  
+Eine Idee, die dabei aufgekommen ist Sharezone als ein Tool für Schüler:innen sowie Lehrkräfte im digitalen Unterricht der Zukunft.  
+
+In iPad-Klassen könnte beispielsweise die Lehrkraft direkt Dateien und Links an Schüler:innen freigeben, Kleingruppen erstellen und die Mitarbeit eintragen.   
+
+Schüler:innen könnten auf ein Blick alle von der Lehrkraft verteilten Dateien sehen, mit Mitschüler:innen kollaborativ an Dokumenten arbeiten und eigene Notizen zum Unterricht schreiben. Wer weiß, vielleicht kann Sharezone ja auch direkt eine Ausrede für vergessene Hausaufgaben finden 😉
+
+Die Mockups unten sind nur schnell zusammengeklebt und ergeben an vielen Stellen noch kein Sinn. Konzentriert euch eher auf die Idee als auf das Aussehen ;)
+
+*Ansicht Schüler:in*
+![iPad-Ansicht von Schüler:innen für den Unterricht]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/aktuelle-stunde-schueler.svg)
+
+*Ansicht Lehrer:in*
+![iPad-Ansicht von Lehrkraft für den Unterricht]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/aktuelle-stunde-lehrer.svg)
+
+Schreibt uns gerne mal eure Meinung zur Grundidee.   
+Das ganze wird wahrscheinlich nichts sein, was bald kommen wird - es ist eher eine Idee für die fernere Zukunft :)
+
+## Sonstiges
+* Wir haben unsere Google-Cloud Startup-Credits noch bis Juli verlängern können! Heißt: Wir können noch länger das Geld für Hosting-Gebühren sparen 🙌
+* Jonas hat sich bei einem Startup-Programm in seiner Heimat Siegen beworben - So können wir hoffentlich auch lokal noch von einem Gründernetzwerk profitieren 👍
```

Comment:
Oben hattest du in der Ich-Perspektive geschrieben und jetzt in der 3. Person 😅

Reply:
Habs gefixt